src/utils/check.your.answers.ts

```diff
@@ -0,0 +1,98 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../utils/logger";
+import { Session } from "@companieshouse/node-session-handler";
+
+import { ApplicationData } from "../model";
+import { getApplicationData } from "../utils/application.data";
+import { isActiveFeature } from "../utils/feature.flag";
+import { createOverseasEntity } from "../service/overseas.entities.service";
+import { OverseasEntityKey, Transactionkey } from "../model/data.types.model";
+import { closeTransaction, postTransaction } from "../service/transaction.service";
+import { startPaymentsSession } from "../service/payment.service";
+
+import {
+  OVERSEAS_ENTITY_UPDATE_DETAILS_URL,
+  OVERSEAS_ENTITY_SECTION_HEADING,
+  WHO_IS_MAKING_UPDATE_URL,
+  FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME,
+  UPDATE_AN_OVERSEAS_ENTITY_URL,
+  CHS_URL
+} from "../config";
+
```

Comment:
I've just noticed these lets are on the global scope. They would be shared by all node requests? Please move into functions. You don't really need the isNoChangeJourney as you can use the noChangeFlag (with undefined being false).